lib/install/install.go

```diff
@@ -205,6 +211,8 @@ type Config struct {
 	NewProcess process.NewGravityProcess
 	// Silent allows installer to output its progress
 	localenv.Silent
+	// LocalClusterClient is a factory for creating client to the installed cluster
+	LocalClusterClient func() (*opsclient.Client, error)
```

Comment:
This needs to be a constructor function instead an object since this is used during installation and the client can only be created once the cluster has been setup.